init-tools.sh

```diff
@@ -109,9 +109,10 @@ if [ ! -e $__INIT_TOOLS_DONE_MARKER ]; then
 
     Tools/crossgen.sh $__scriptpath/Tools
 
+    mkdir -p "$(dirname "$__INIT_TOOLS_DONE_MARKER")"
```

Comment:
Isn't the folder always created at this point when the tools initialization succeeded? It looks like it should, since it also contains project.json file for build tools dependencies.